polytope/polytope.py

```diff
@@ -78,7 +78,10 @@
     logger.warn(
         '`polytope` failed to import `cvxopt.glpk`.\n'
         'Will use `scipy.optimize.linprog`.')
-
+try:
+    xrange
```

Comment:
If "go through and replace" refers to replacing in code `xrange` with `range`, then in Python 2 we would get inefficient behavior, so that would suggest that Python 3 is intended by default, and Python 2 for the exceptional user that hasn't updated to Python 3 yet. `tulip` is still in Python 2 though, so we cannot make this assumption yet. When that changes (https://github.com/tulip-control/tulip-control/issues/44 -- my intention is before 2017), then we can substitute `range` for `xrange` and deprecate Python 2, remaining compatible until its support expires.